src/Restaurant.js

```diff
@@ -1,17 +1,23 @@
-import React from "react";
+import React, { Component } from "react";
+import Orders from "./Orders"
 
-const Restaurant = () => {
-  const pizzas = 0;
-  return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
-      </ul>
-    </div>
-  );
-};
+class Restaurant extends Component {
+
+
+  render() {
+        return (
+      <div>
+        <h3>Restaurant Orders</h3>
+        <ul>
+          <Orders orderType="Salads" />
+          <br/>
+          <Orders orderType="Pizzas" />
```

Comment:
Can you see how making our components reusable (but configurable via props) is useful? :)